service/installer/helm/helm.go

```diff
@@ -188,6 +213,47 @@ func (i *HelmInstaller) runHelmCommand(name string, args ...string) error {
 	return nil
 }
 
+func (i *HelmInstaller) fetchMetrics() error {
+	ticker := time.NewTicker(i.pollInterval)
+	projectList := project.GetProjectList(i.provider, i.environment)
+
+	go func() {
+		for c := ticker.C; ; <-c {
+			i.logger.Log("debug", "fetching metrics")
+			i.checkHelmRelease(projectList)
+			i.logger.Log("debug", "fetched metrics")
+		}
+	}()
+
+	return nil
+}
+
+func (i *HelmInstaller) checkHelmRelease(projectList []string) {
+	for _, prj := range projectList {
+		cmd := exec.Command(i.helmBinaryPath, "history", prj, "--output", "yaml", "--max", "1")
+
+		var stdOutBuf, stdErrBuf bytes.Buffer
+		cmd.Stdout = &stdOutBuf
+		cmd.Stderr = &stdErrBuf
+
+		err := cmd.Run()
+		if err != nil {
+			i.logger.Log("error", fmt.Sprintf("could not find helm release %#q history", prj), stdErrBuf.String())
+		}
+
+		var v []map[string]string
+		err = yaml.Unmarshal(stdOutBuf.Bytes(), &v)
```

Comment:
This is the critical point, we get `helm history` as YAML output and parsing again to get `status` from the release.